TESTING.md

```diff
@@ -321,3 +54,14 @@ api:
   password: pass
   port: 8000
 ```
+
+
+## Useful commands
+
+Create unbind commands for all bindings:
+
+  ./gcp-service-broker show bindings | jq --raw-output '.[] | "gcp-service-broker client unbind --bindingid \(.BindingId) --instanceid \(.ServiceInstanceId) --planid \(.PlanId) --serviceid \(.ServiceId)"'
```

Comment:
Would it make sense to bypass the copy/paste step for the user and just execute the commands with xargs?

Reply:
I like that idea, so far I've had to slightly tweak the outputs so they use the same configuration options as the current command (`--config ../minimal.yml`).

What do you think about a `./gcp-service-broker client unbind-all` command? I keep wavering about it potentially being too dangerous but it would be really handy.